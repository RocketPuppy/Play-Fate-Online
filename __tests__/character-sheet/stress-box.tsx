import * as React from "react";
import * as renderer from "react-test-renderer";
import StressBox from "../../src/character-sheet/stress-box";

const noop = () => {};

test("renders with the stress level", () => {
  const render = renderer.create(<StressBox stressLevel={2} used={false} onUse={noop} />);

  expect(render.toJSON()).toMatchSnapshot();
});

test("renders unchecked", () => {
  const render = renderer.create(<StressBox stressLevel={2} used={false} onUse={noop} />);

  expect(render.toJSON()).toMatchSnapshot();
});


test("renders checked", () => {
  const render = renderer.create(<StressBox stressLevel={2} used={true} onUse={noop} />);

  expect(render.toJSON()).toMatchSnapshot();
});
