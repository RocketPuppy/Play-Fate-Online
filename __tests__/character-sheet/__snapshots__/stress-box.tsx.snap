// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders checked 1`] = `
<label>
  <span>
    2
  </span>
  <input
    checked={true}
    onChange={[Function]}
    type="checkbox"
  />
</label>
`;

exports[`renders unchecked 1`] = `
<label>
  <span>
    2
  </span>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
</label>
`;

exports[`renders with the stress level 1`] = `
<label>
  <span>
    2
  </span>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
</label>
`;
